feature/experimental/isis/otg_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -91,9 +110,20 @@ func TestMetric(t *testing.T) {
 	if err := check.Equal(metric.State(), uint32(100)).AwaitFor(time.Second, ts.DUTClient); err != nil {
 		t.Error(err)
 	}
-	// TODO: Verify the link state database on the ATE once the ATE reports this properly
-	// ateTelemPth := ts.ATEISISTelemetry(t)
-	// ateDB := ateTelemPth.Level(2).LspAny()
-	// for _, nbr := range ateDB.Tlv(telemetry.IsisLsdbTypes_ISIS_TLV_TYPE_IS_NEIGHBOR_ATTRIBUTE).IsisNeighborAttribute().NeighborAny().Get(t) {
-	// }
+
+	_, ok := gnmi.WatchAll(t, otg, gnmi.OTG().IsisRouter("devIsis").LinkStateDatabase().LspsAny().Tlvs().ExtendedIpv4Reachability().PrefixAny().Metric().State(), time.Minute, func(v *ygnmi.Value[uint32]) bool {
+		time.Sleep(1 * time.Second)
```

Comment:
Last thing :) : Can we remove this one as well?

Reply:
Done